packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -112,13 +121,10 @@ export const fetchTemplateZipFromLocalAction: GeneratorAction = {
     if (context.zip) {
       return;
     }
-
-    if (!context.fallbackZipPath) {
-      context.fallbackZipPath = path.join(getTemplatesFolder(), "fallback");
-    }
-
+    context.fallback = true;
+    const fallbackPath = path.join(getTemplatesFolder(), "fallback");
```

Comment:
What's the reason behind this modification? I am not convincd that it improves the code. By hard-coding the fallback path in this action, we are limiting our ability to customize it in the future.

Reply:
The reason is to try to make generatorContext concise. The original generatorContext.fallbackPath's value is never actually used after set and it's purpose is just to send telemetry, so we can change it to be a boolean.